apps/daimo-mobile/src/view/shared/ContactBubble.tsx

```diff
@@ -103,11 +114,25 @@ export function Bubble({
     [size, col]
   );
 
+  const imageStyle: ImageStyle = useMemo(
+    () => ({
+      position: "absolute",
+      height: size,
+      width: size,
+      borderRadius: size / 2,
+      zIndex: 5,
+    }),
+    [size]
+  );
+
   return (
-    <View style={style}>
-      <Text style={textStyle} numberOfLines={1} allowFontScaling={false}>
-        {children}
-      </Text>
+    <View style={{ width: size, height: size }}>
+      <Image source={image} style={imageStyle} />
+      <View style={style}>
```

Comment:
do we want both the image and the letter? i thought just one or the other

Reply:
In this case, the letter is the fallback. If we make it one or the other, a broken image (e.g. no PFP) means that we get a blank view. Using `<Image>`'s `onError` prop to set a "failed image fetch" state also doesn't work, since it waits until subsequent renders to update to the text.